spec/models/ontology_spec.rb

```diff
@@ -147,18 +147,53 @@
       end
     end
 
-    context 'an imported ontology' do
-      let(:ontology) { create :ontology }
+    context 'an imported ontology (meaning any kind of mapping)' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.